support-frontend/yarn.lock

```diff
@@ -3085,6 +3085,18 @@
     telejson "^3.2.0"
     util-deprecate "^1.0.2"
 
+"@stripe/react-stripe-js@^1.1.2":
```

Comment:
Is this yarn.lock update needed for this PR? It doesn't look like any dependencies have changed.

Reply:
I had to `yarn install` when I switched back to main as the react-stripe dependency had been added by another PR. I'd assumed it was just some minor dependency bumps but it looks like this dependency wasn't in the lockfile at all before, which is weird as it seems to have been added months ago: https://github.com/guardian/support-frontend/blame/main/support-frontend/package.json#L87 🤔

Reply:
That is odd, maybe the lock file hasn't been updated since it was added? 😕 
I'm happy to leave this change in though.